packages/api-server/package.json

```diff
@@ -37,7 +38,7 @@
   },
   "scripts": {
     "build": "yarn build:js",
-    "prepublishOnly": "yarn cross-env NODE_ENV=production yarn build",
+    "prepublishOnly": "NODE_ENV=production yarn build",
```

Comment:
@peterp do you know when/how this script is currently being used? My assumption is that it was added for Lerna publishing command — but the reality is I don't really know.

Reply:
It's called just before you run `npm publish`. The reason why we added it is no longer known to me, but I can assume that it's for safeties sake. So that we don't accidentally publish an empty package.